packages/action-menu/src/ActionMenu.ts

```diff
@@ -43,6 +43,9 @@ export class ActionMenu extends ObserveSlotText(PickerBase, 'label') {
     @property({ type: String })
     public override selects: undefined | 'single' = undefined;
 
+    @property({ type: String, reflect: true })
+    public static: 'white' | 'black' | undefined = undefined;
```

Comment:
default fallback should be 'none' instead of undefined. We should not show user undefined in the select dropdown in storybook

Reply:
done!

Reply:
done!